resdk/resolwe.py

```diff
@@ -267,7 +267,7 @@ def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined
         :param dict input: Input values
         :param dict descriptor: Descriptor values
         :param str descriptor_schema: A valid descriptor schema slug
-        :param list collection: Collection into which data object should be included
+        :param int/resource collection: Collection resource or it's id into which data object should be included
```

Comment:
Docstrings should be wrapped at 72 characters.

Reply:
Done.